packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -1,46 +1,46 @@
-import { createStackNavigator, createSwitchNavigator } from 'react-navigation'
-import Account from 'src/account/Account'
-import Analytics from 'src/account/Analytics'
-import DollarEducation from 'src/account/DollarEducation'
-import EditProfile from 'src/account/EditProfile'
-import GoldEducation from 'src/account/GoldEducation'
-import Invite from 'src/account/Invite'
-import InviteReview from 'src/account/InviteReview'
-import Licenses from 'src/account/Licenses'
-import PhotosEducation from 'src/account/PhotosEducation'
-import Profile from 'src/account/Profile'
-import AppLoading from 'src/app/AppLoading'
-import Debug from 'src/app/Debug'
-import ErrorScreen from 'src/app/ErrorScreen'
-import UpgradeScreen from 'src/app/UpgradeScreen'
-import Backup from 'src/backup/Backup'
-import DappKitAccountScreen from 'src/dappkit/DappKitAccountScreen'
-import DappKitTxSignScreen from 'src/dappkit/DappKitTxSignScreen'
-import ReclaimPaymentConfirmationScreen from 'src/escrow/ReclaimPaymentConfirmationScreen'
-import ExchangeReview from 'src/exchange/ExchangeReview'
-import ExchangeTradeScreen from 'src/exchange/ExchangeTradeScreen'
-import ImportContacts from 'src/import/ImportContacts'
-import ImportWallet from 'src/import/ImportWallet'
-import EnterInviteCode from 'src/invite/EnterInviteCode'
-import JoinCelo from 'src/invite/JoinCelo'
-import Language from 'src/language/Language'
-import { Screens, Stacks } from 'src/navigator/Screens'
-import TabNavigator from 'src/navigator/TabNavigator'
-import PaymentRequestListScreen from 'src/paymentRequest/PaymentRequestListScreen'
-import Pincode from 'src/pincode/Pincode'
-import PincodeConfirmation from 'src/pincode/PincodeConfirmation'
-import QRCode from 'src/qrcode/QRCode'
-import QRScanner from 'src/qrcode/QRScanner'
-import FeeEducation from 'src/send/FeeEducation'
-import RequestConfirmation from 'src/send/RequestConfirmation'
-import SendAmount from 'src/send/SendAmount'
-import SendConfirmation from 'src/send/SendConfirmation'
-import SetClock from 'src/set-clock/SetClock'
-import TransactionReviewScreen from 'src/transactions/TransactionReviewScreen'
-import VerifyEducation from 'src/verify/Education'
-import VerifyInput from 'src/verify/Input'
-import VerifyVerified from 'src/verify/Verified'
-import VerifyVerifying from 'src/verify/Verifying'
+import { createStackNavigator, createSwitchNavigator } from 'react-navigation';
```

Comment:
I think this is a sign that your vs code isn't configured right? 
I'm surprised prettier didn't remove these semicolons for you. Please fix.